docs/runtime_environment.rst

```diff
@@ -56,3 +56,11 @@ Note that non-relative imports to these files will `not` work:
 
    import backend
    from utils import img_utils, ml_utils
+
+Limiting GPU memory Growth
+-----------------------------------------
+
+By default Vcap maps all available memory of all visible CUDA configured GPUs,
```

Comment:
Minor thing, but when referring to the system in the docs, we usually call it "OpenVisionCapsules" instead of "vcap". "vcap" is just the name of the Python package.